src/main/java/com/denizenscript/clientizen/objects/ModTag.java

```diff
@@ -0,0 +1,251 @@
+package com.denizenscript.clientizen.objects;
+
+import com.denizenscript.clientizen.util.Utilities;
+import com.denizenscript.denizencore.events.ScriptEvent;
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import net.fabricmc.loader.api.FabricLoader;
+import net.fabricmc.loader.api.ModContainer;
+import net.fabricmc.loader.api.metadata.ModMetadata;
+
+import java.util.Optional;
+
+public class ModTag implements ObjectTag {
+
+    // <--[ObjectType]
+    // @name ModTag
+    // @prefix mod
+    // @base ElementTag
+    // @ExampleTagBase mod[clientizen]
+    // @ExampleValues <mod[clientizen]>
+    // @ExampleForReturns
+    // - narrate "The mod is %VALUE%!"
+    // @format
+    // The identity format for mods is the mod id.
+    // For example, 'mod@clientizen' or 'mod@theprinter'.
+    //
+    // @description
+    // A ModTag represents a currently loaded fabric mod.
```

Comment:
fabric -> Fabric 

Or you might want to pre-emptively remove this qualifier entirely and just say "loaded mod" in case Clientizen gets ported to other loaders

Reply:
Removed it for now